### ISSUE TITLE
How to modify the codes

### ISSUE THREAD

**Issue by BearVic** [fizyr/keras-retinanet#625]
I want to modify some codes in ``filter_detection`` of  ``layers``, however, no matter how I change the codes, it does not change the results. 

Is it because of the instruction of  ``pip install . --user`` that install the whole repository as a public package? How to uninstall? 

I googled and tried ``pip uninstall .`` but did not work. I also tried to run the alternative instruction ``python setup.py build_ext --inplace`` and it did not work, neither. Please help me figure it out since I've been stuck here for a long time. Thanks!

**Comment by mukeshmithrakumar** [comment]
What i did was I just downloaded the repo, did not install, changed the imports to absolute based on my directory structure and modified individual modules, that worked for me and as to why your modification doesn't work, I am not sure if its because you installed the whole repo but it shouldn't matter

**Comment by yhenon** [comment]
Closing, as this is general python usage, and has been answered on other forums such as StackOverflow.

**Comment by BearVic** [comment]
@mukeshmithrakumar I tried that too. I re-download a repo and changed the imports to the absolute path but did not work. I think it is because I've already ``pip install``. To solve this problem, we can remove files in ``~/.local/bin/``. It works for me. @yhenon Sorry to ask the stupid question. I googled before but not carefully enough. I tried two methods in the stackoverflow but there is a third one. Thanks!
